v20.1/scalar-expressions.md

```diff
@@ -727,16 +713,42 @@ For example:
 ~~~
 
 ~~~
-+----------------+
-|       a        |
-+----------------+
-| ('x',123,12.3) |
-+----------------+
+       a
+----------------
+  (x,123,12.3)
+(1 row)
 ~~~
 
 The data type of the resulting tuple is inferred from the values.
 Each position in a tuple can have a distinct data type.
 
+<span class="version-tag">New in v20.1:</span> CockroachDB supports accessing the Nth element in a tuple as a single table cell using the syntax `(...).@N`. For example:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT (t).@2 FROM (SELECT (1,'b',2.3) AS t);
+~~~
+
+~~~
+  ?column?
+------------
+  b
+(1 row)
+~~~
+
+CockroachDB also supports expanding all elements of a tuple as a single row with the `.*` notation. For example:
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT (t).* FROM (SELECT (1,'b',2.3) AS t)
```

Comment:
maybe use an example where the values in the table are named, that will show how the column names are derived from the tuple labels